source/tests/system/nidcpower_driver_api_tests.cpp

```diff
@@ -349,6 +407,32 @@ TEST_F(NiDCPowerDriverApiTest, NiDCPowerSetViInt64Attribute_SendRequest_GetViInt
   EXPECT_EQ(expected_value, get_attribute_value);
 }
 
+TEST_F(NiDCPowerDriverApiTest, VoltageLevelConfiguredAndExportedToBuffer_ResetAndImportConfigurationFromBuffer_ConfigurationIsImportedSuccessfully)
+{
+  const char* channel_name = "0";
+  ViReal64 expected_voltage_level = 3.0;
+  configure_output_function(channel_name, dcpower::OutputFunction::OUTPUT_FUNCTION_NIDCPOWER_VAL_DC_VOLTAGE);
+  configure_voltage_level(channel_name, expected_voltage_level);
+  auto export_buffer_response = export_attribute_configuration_buffer();
+
+  reset_with_channels(channel_name);
+  ::grpc::ClientContext context;
+  dcpower::ImportAttributeConfigurationBufferRequest request;
+  request.mutable_vi()->set_id(GetSessionId());
+  for (int i = 0; i < export_buffer_response.configuration_size(); i++)
+  {
+    auto configuration = export_buffer_response.configuration(i);
+    request.add_configuration(configuration);
+  }
+  dcpower::ImportAttributeConfigurationBufferResponse response;
+  ::grpc::Status status = GetStub()->ImportAttributeConfigurationBuffer(&context, request, &response);
+  EXPECT_TRUE(status.ok());
+  expect_api_success(response.status());
+
+  ViReal64 actual_voltage_level = get_real64_attribute(channel_name, dcpower::NiDCPowerAttributes::NIDCPOWER_ATTRIBUTE_VOLTAGE_LEVEL);
```

Comment:
Does it make sense to also verify the output function was set by getting the `NIDCPOWER_ATTRIBUTE_OUTPUT_FUNCTION` attribute?

Reply:
Done adding that assert